src/odemis/gui/cont/temperature.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+
+"""
+@author: Anders Muskens
+
+Copyright © 2020 Anders Muskens, Delmic
+
+Handles the switch of the content of the main GUI tabs.
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License version 2 as published by the Free
+Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+"""
+
+import wx
+import collections
+import numpy
+import time
+import logging
+from odemis.gui.util import call_in_wx_main
+import odemis.gui as gui
+from odemis.gui.comp import popup
+import odemis.model as md
+
+MIN_PERIOD_WARNING = 60
+
+
+@call_in_wx_main
+def set_label_colour(label, colour, bold):
+    """
+    Set a wx.StaticText foreground colour and font style to bold/unbold
+    label: wx.StaticText object
+    colour: wx.Colour object
+    bold: (bool) true to set to bold
+    """
+    label.SetForegroundColour(colour)
+    if bold:
+        font = label.GetFont()
+        font.SetWeight(wx.BOLD)
+        label.SetFont(font)
```

Comment:
You can even write `font.SetWeight(wx.BOLD if bold else wx.NORMAL)`, and make the whole thing just 3 lines ;-)